### ISSUE TITLE
Adding own Hotspot images

### ISSUE THREAD

**Issue by 4pr3x** [h5p/h5p-image-hotspots#8]
Hey,

first of all, thanks for the great plugins. It works really good. 

I would like to be able to add my own icons for the clickable hotspots. Best would be if this will be sometime integrated into the UI. 

For the meantime, could you give me a Hint how/ where the current Image for the hotspot is created/ loaded. So I could try to change it on my own in the code, but haven´t found the right place yet.
I found where you set the color of the spot and I know that you use FontAwesome for the icon, but I couldn´t find where you specified the css class 'fa-plus-circle'.

Thanks in Advance.

**Comment by 4pr3x** [comment]
Found it on my own. Search for some reason for the css class, but you are using content values. :)

**Comment by thomasmars** [comment]
Indeed, it is an icon and it uses the unicode "f055" at https://github.com/h5p/h5p-image-hotspots/blob/master/styles/image-hotspots.css#L46. A pull request would have to be created for it to be integrated into the UI :)

**Comment by y0x** [comment]
I need that feature too. I´m currently working on it, but it seems to be slightly painful to override the pseudo selector. :/ 

@4pr3x I let you know when it´s done/ will make a PR.

**Comment by y0x** [comment]
@thomasmars 

I´m thinking about the UI integration. My current idea is to make a textfield, where the user can put in the unicode. A link to fontawesome.io in the description field explaining the user where to find the codes. 

What do you think about this? Any other idea for the UI?

**Comment by thomasmars** [comment]
@y0x 
I think this is an okay solution for users that are used to css and coding, however I don't think it is very user friendly.

A different route could be to create an editor widget that would render a subset of FontAwesome icons into a dropdown or select box where you could visually just select the icon that you wanted from the editor. The widget would then make sure that this value is stored semantically similar to the way you suggested, but the user would not have to deal with this part of the logic. This would of course require a bit more work to implement.

For the implementation in the view I would suggest finding the element, then apply the unicode of the fontawesome icon to the appropriate pseudo element ```content``` property. This could be accomplished through JS, as the unicode would be passed to the content type as a parameter.
